### PR TITLE
add unit test for NodeWightSlice2StringSlice

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: dfget/config/supernode_value_test.go [hunk 1]
```diff
@@ -98,3 +98,14 @@ func (suit *SupernodeValueSuite) TestHandleNodes() {
 		}
 	}
 }
+
+func (suit *SupernodeValueSuite) TestNodeWightSlice2StringSlice() {
```
Comment (allencloud) [inline dfget/config/supernode_value_test.go]: I think the `Wight` should be the word `Weight`, do you think so ? @Starnop @lowzj 
If that, would change all the `Wight` into `Weight`in file https://github.com/dragonflyoss/Dragonfly/blob/b32d1ca03b4111978b2d8a0f398704946e071d99/dfget/config/supernode_value.go and some other files?

Comment (q384566678) [reply dfget/config/supernode_value_test.go]: I think this should be a typo, it should be Weight, not Wight.

Comment (allencloud) [reply dfget/config/supernode_value_test.go]: Could you add a pull request to change them all?

Comment (q384566678) [reply dfget/config/supernode_value_test.go]: Done, PTAL.

Comment (allencloud) [reply dfget/config/supernode_value_test.go]: Thank you so much.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**